src/lua/uri.lua

```diff
@@ -272,39 +305,6 @@ local function values(...)
     return {...}
 end
 
--- Encodes a string into its escaped hexadecimal representation.
```

Comment:
PR #7254 doesn't land ATM, so we can place the functions at needed place from scratch. Or use a forward declaration.

```lua
local escape

local function foo()
   <..use escape..>
end

escape = function(<...>)
    <...>
end
```

Feel free to ignore, but in this case I beg you to give extra care at future rebase, because it may leave old version of the code after the movement. The conflict detection will not complain against the added code, only about removed.